scripts/junit_report.js

```diff
@@ -7,8 +7,6 @@ function uploadJUnitXMLReport () {
     console.log('Running in a fork. Skipping step.')
     return
   }
-  // we install @datadog/datadog-ci
-  execSync('yarn global add @datadog/datadog-ci@0.13.2', { stdio: 'inherit' })
```

Comment:
I don't remember, why was this added in this way?

Reply:
I think not to have it as a dependency